.github/workflows/docker-hub.yml

```diff
@@ -109,7 +110,10 @@ jobs:
           context: .
           file: ./src/frontend/Dockerfile
           target: frontend-production
-          build-args: DOCKER_USER=${{ env.DOCKER_USER }}:-1000
+          build-args: |
+            DOCKER_USER=${{ env.DOCKER_USER }}:-1000
+            SENTRY_DSN=${{ env.SENTRY_DSN }}
+            SENTRY_ENV=${{ github.event_name != 'pull_request' && github.ref == 'refs/heads/main' && 'production' || 'staging' }}
```

Comment:
I don't think this should be set in the CI. Sentry is to monitor our envs.

Reply:
I don't see another way for the moment to distinguish the environment on the frontend side. I talk about it with Jacques, we wanted maybe to add a `config.json` for the frontend (mount point), and add these vars in it, like that we could know if the logs are from `staging` / `preprod` / `production`.